### ISSUE TITLE
A slider component

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami#13]
## What

A [range input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range). 

## Why

There is an opportunity to reduce repeated work and improve consistency: 

- ["IG use these a lot"](https://github.com/Financial-Times/ft-origami/issues/533)
- Lodestar uses a range slider.

## Supporting Examples

https://ig.ft.com/sites/isa-calculator/
<img width="770" alt="screenshot 2019-02-28 at 15 23 53" src="https://user-images.githubusercontent.com/10405691/53577051-e0785500-3b6c-11e9-8288-1dd446673b8d.png">

https://lodestar.ft.com/
<img width="460" alt="screenshot 2019-02-28 at 15 21 42" src="https://user-images.githubusercontent.com/10405691/53577071-ec641700-3b6c-11e9-9e16-7f2c8a24f3d5.png">

**Comment by JakeChampion** [comment]
@notlee Do the relevant people in IG and IP know that we are interested in doing this?

**Comment by notlee** [comment]
No, I was just migrating this issue to the correct place. We should talk to them as part considering a slider :)

**Comment by chee** [comment]
https://api.github.com/repos/Financial-Times/origami/pulls/97
